packages/universal-xml-plugin/lib/xpath.js

```diff
@@ -1,10 +1,19 @@
 import {select as xpathQuery} from 'xpath';
-import {DOMParser, MIME_TYPE} from '@xmldom/xmldom';
+import {DOMParser, MIME_TYPE, Document} from '@xmldom/xmldom';
 import _ from 'lodash';
 
+/**
+ * @param {Document} doc - The Document object to be converted.
+ * @returns {Node} - The converted Node object.
+ */
+function convertDocumentToNode(doc) {
+  // @ts-ignore
```

Comment:
why ignore is needed?

Reply:
I'm also not quite sure if that's not a bug in xmldom itself.

Reply:
I've reported it as https://github.com/xmldom/xmldom/issues/724

Reply:
thanks